offre_realisee/infrastructure/local_file_system_handler.py

```diff
@@ -147,6 +157,10 @@ def get_daily_mesure_qs(self, date: datetime, mesure_type: MesureType,
         folder_path = os.path.join(
             self.data_path, self.output_path, AggregationLevel.by_day, mesure_type
         )
+        if dsp is not None:

```

Comment:
"" is not None est True aussi non ?

Reply:
Top, bien vu, je pensais les avoir tous eu.